app/domain/logs.rb

```diff
@@ -555,6 +555,41 @@ module AuthnJwt
         code: "CONJ00109D"
       )
 
+      FetchingIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching identity path...",
+        code: "CONJ00110D"
+      )
+
+      FetchedIdentityPath = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully fetched identity path '{0-identity-path}'",
+        code: "CONJ00111I"
+      )
+
+      FetchingIdentityByInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching identity by interface: '{0-interface-name}'",
+        code: "CONJ00112D"
+      )
+
+      FetchedIdentityByInterface = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully fetched identity '{0-identity}' by interface: '{1-interface-name}'",
+        code: "CONJ00113I"
+      )
+
+      AddingIdentityPrefixToIdentity = ::Util::TrackableLogMessageClass.new(
+        msg: "Adding identity prefix '{0-identity-prefix}' to identity '{1-identity}'...",
```

Comment:
Example, please?

Reply:
identity-path is a variable in the authenticator policy which defined the host identity path(location in the policy tree),
in the following example:
the identity in the token is: 00uksxz3o2vFJCX230h7
the customer configured the identiy-path to hosts in path /apps/
the host definition in Conjur is : 'host/apps/00uksxz3o2vFJCX230h7

```
 CONJ00111I Successfully fetched identity path 'apps'
 CONJ00116D Adding identity prefix 'apps' to identity '00uksxz3o2vFJCX230h7'...
 CONJ00117D Successfully added identity prefix, calculated identity is: 'apps/00uksxz3o2vFJCX230h7'
 CONJ00116D Adding identity prefix 'host' to identity 'apps/00uksxz3o2vFJCX230h7'...
 CONJ00117D Successfully added identity prefix, calculated identity is: 'host/apps/00uksxz3o2vFJCX230h7'
 CONJ00115I Successfully fetched identity 'host/apps/00uksxz3o2vFJCX230h7' by interface: 'token-identity-provider'

 CONJ00098I Successfully found JWT identity 'host/apps/00uksxz3o2vFJCX230h7'

```

Reply:
fine